develop/how-to-report-issues/index.md

```diff
@@ -0,0 +1,83 @@
+
+Making the patch is trivial if you use git: then you just need to run
+```
+git diff origin/master.. > repro-for-my-bug.patch
+```
+
+If you don't use git, please use the standalone `diff` program with the `-u` option to create a unified diff. The most general form of the diff command you need to use is
+```
+diff -uNr wxWidgets-orig wxWidgets-mine > repro-for-my-bug.patch
+```
+
+but you can omit `-N` and `-r` parts if you have modified a single file only. However please still run diff from the top level directory and **not** from the sample subdirectory to make applying it easier.
+
+To summarize, here are the salient points once again:
+
+1. You should make a single patch with all changes and make from the top of wxWidgets tree.
+2. This patch should be in unified diff format.
+3. Please attach the patch to the issue you create.
+
+Once again, please notice that providing a way to reproduce the bug is a vital first step in fixing it. It doesn't guarantee that the bug will be fixed, but if we're not able to reproduce the bug, it does almost guarantees that we will not fix it neither. Thanks in advance for your help!
+
+## Fatal Bugs
+
+In a special case of reporting a fatal bug/crash please run the program under the debugger and include the stack trace at the moment of the crash in your report.
+
+Under Unix it is as simple as running `gdb your-program-name` (under OS X, use `gdb your-app/Contents/MacOS/your-app`), typing `r` in gdb and then typing `bt` when the crash happens. You may do it even with non-debug build of wxWidgets as this should still give us at least the names of the functions but ideally you should rebuild wxWidgets with debug information as this would give the line numbers and parameter values as well.
+
+Under Windows you should use a debug build and run the program under Microsoft Visual Studio debugger if you're using MSVC compiler or under WinDBG otherwise.
+
+## Creating the Issue
+
+* Title: Please provide a meaningful title for your issue.
+
+* Priority: A special note about priority labels: you may choose to use lower priority for the bug but in general please ***do not set high*** priority, even if the bug is very important to you personally. We will increment the priority ourselves if the bug is deemed to be more serious than normal from wxWidgets point of view.
+
+* Version: Please use the appropriate label for the version. If you've tested multiple versions, please mention this in the main text.
+
+* Other labels: Please select any labels that are appropriate, but make sure to use the label corresponding to the port (e.g. "MSW" or "GTK") for which you're reporting the issue. If the bug is in the `base` sub-library, select `base`. Note that `build` label is for ticket related to bakefiles, `configure.in`, `wx-config`, etc, please use specific label for CMake. Finally, please use `regression` label for anything that used to work in a previous wxWidgets version but doesn't work any longer.
```

Comment:
"Note that `build` label is for ticket related to bakefiles, `configure.in`, `wx-config`, etc, ..."
That is rather Linux specific, is it not? I have created many build issues yet I do not use any of those. Or should there be "**m**akefiles" instead of "bakefiles" (which are probably rare touched)?